pkg/scheduler/framework/plugins/volumebinding/binder.go

```diff
@@ -884,6 +884,12 @@ func (b *volumeBinder) checkBoundClaims(logger klog.Logger, claims []*v1.Persist
 			return false, true, err
 		}
 
+		err = volume.CheckCSINodeDriverAffinity(csiNode, pv)
+		if err != nil {
+			logger.V(4).Info("PersistentVolume and csinode driver mismatch for pod", "PV", klog.KRef("", pvName), "csinode", klog.KObj(csiNode), "pod", klog.KObj(pod), "err", err)
+			return false, true, nil
```

Comment:
Is this right to return nil error here?

Reply:
yep, it possesses identical logic to the CheckNodeAffinity function shown below.

Reply:
If so, then how will the schedule failure message be exposed to user?